templates/careers/departments.html

```diff
@@ -19,76 +19,242 @@ <h1 class="p-heading--2">Departments at Canonical</h1>
 </section>
 
 <section class="p-strip is-bordered">
-  <div class="row">
-    <div class="col-4 u-sv3">
-      <h2>Engineering</h2>
-      <p>Open source is transforming the entire stack. This is your chance to be right at the centre of that revolution, to shape the platforms and tools that millions use to invent our global technology future. From bare metal to cloud and high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
-      <a href="/careers/all?filter=Engineering">Check out open Engineering roles</a>
+  <div class="p-strip is-shallow">
+    <div class="row">
+      <hr>
+      <div class="col-6">
+        <h2>Engineering</h2>
+        <p>Open source is transforming the entire stack. This is your chance to be right at the centre of that revolution, to shape the platforms and tools that millions use to invent our global technology future. From bare metal to cloud and high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
```

Comment:
According to design, this paragraph needs to be divided. 
<img width="383" alt="image" src="https://user-images.githubusercontent.com/57550290/197984772-8538ee59-da40-46f8-bff0-c3b294fa793e.png">

```suggestion
        <p>Open source is transforming the entire stack. This is your chance to be right at the centre of that revolution, to shape the platforms and tools that millions use to invent our global technology future. </p>
        <p>From bare metal to cloud and high performance computing, from AI and big data to the web and connected devices, open source is the key ingredient for success.</p>
```